lib/Twig/Test/IntegrationTestCase.php

```diff
@@ -48,6 +82,11 @@ public function getTests($name, $legacyTests = false)
             }
 
             if ($legacyTests xor false !== strpos($file->getRealpath(), '.legacy.test')) {
+                if (empty($tests)) {
+                    // add a dummy test to avoid a PHPUnit message
+                    $tests[] = array('not', '-', '', array(), '', array());
```

Comment:
this should be added only at the end of the loop. We don't want to add tons of dummy tests